blocks/identity-block/components/EditableFormInputField/index.jsx

```diff
@@ -58,10 +47,13 @@ function EditableFieldPresentational({
   children,
   editText,
   onSubmit,
+  cancelEdit,
+  formErrorText,
 }) {
+  const [isEditable, setIsEditable] = useState(!!formErrorText);
   const { arcSite } = useFusionContext();
-  const primaryColor = getThemeStyle(arcSite)['primary-color'];
-  const [isEditable, setIsEditable] = useState(false);
+  const { locale } = getProperties(arcSite);
```

Comment:
I was trying to leave as much fusion hooks out of the components folder. I may try to pass in this phrases with the save text

Reply:
updating now